packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -160,6 +171,8 @@ class IMobileDevice {
       );
       if (result.exitCode == 255 && result.stdout != null && result.stdout.contains('No device found'))
         throw IOSDeviceNotFoundError('ideviceinfo could not find device:\n${result.stdout}');
```

Comment:
I bet the "could not find device" error became more common once iOS 11.4.1 introduced USB restriction mode, which hides the phone as a mux device when it's plugged in but not unlocked.
If you're going to add 'Is the device asking to "Trust this Computer"' you may want to add 'Try unlocking any attached devices.'  I'm also cool adding this change myself in another PR if it's out of scope.

Reply:
Right now, the user only sees the message if they run `flutter -v doctor -v`, and it's buried in a ton of output.  What I'd _really_ like is for us to return a device in their device list that shows as a "not yet ready to use" device, with more info.  Right now, doctor doesn't have that ability.  I think that can be a follow-up change.  In the meantime, I'll update the verbiage of this message.

Reply:
I actually have a draft of that https://github.com/flutter/flutter/pull/36902/files which was getting complicated so I put it on the backburner. I will rebase that on this change. Also, feel free to take this up also, if you feel so inclined.